models/demos/falcon7b/tt/model_config.py

```diff
@@ -129,6 +177,22 @@ def get_model_config(model_config_str, prefill_seq_len=0):
     model_config.update({f"{key}_MEMCFG": mem_config for key in OP_KEYS if key not in NO_MEMCFG})
     model_config.update({f"{key}_DTYPE": dtype for key in OP_KEYS if key not in NO_DTYPE})
 
+    hidden_dim = 4544
+    decode_batch_size = 32
```

Comment:
Can this be a parameter? Since the batch size might change in the future

Reply:
Kk, will do it on Monday

Reply:
Done, added it as part of model_config init